Digipost.Signature.Api.Client.Direct/DirectClient.cs

```diff
@@ -78,6 +81,47 @@ private static Uri RelativeUrl(Job job)
             }
         }
 
+        public async Task<JobStatusResponse> GetStatusChange(Sender sender = null)
```

Comment:
It is more or less duplicate code, however there are some differences in how the code handle the different response codes, i.e. different type to return when receiving no content and deserialization function when receiving actual changes.
I considered refactoring this in to a method both clients could use, but it felt clunky.

Can add some doc.